release-content/0.15/release-notes/_release-notes.toml

```diff
@@ -429,3 +429,10 @@ authors = ["@cart"]
 contributors = ["@alice-i-cecile", "@killercup", "@BD103"]
 prs = [14791]
 file_name = "14791_Required_Components.md"
+
+[[release_notes]]
```

Comment:
Double check that we don't have a duplicate in here: the stub should already have been generated.